### DISCUSSION TITLE
Will you consider adding deep cloning method?

### DISCUSSION THREAD

**Discussion by coderwyd** [radashi-org/radashi#55] [Ideas]
Because `radash` doesn't support that method, which dissuaded a lot of people from

**Comment by aleclarson** [comment]
This is a high priority for sure!

Are there any other functions you'd like to see?

**Comment by aleclarson** [comment]
This is in-progress over at #81

**Comment by aleclarson** [comment]
It's been added. For now, it's available in `radashi@beta` until I get the docs running.
